### PR TITLE
Include version in validation info outputs

### PR DESCRIPTION
**Pull request by JRosenkranz**
This PR builds on https://github.com/foundation-model-stack/aiu-fms-testing-utils/pull/115 and includes utilities for getting validation info paths which include version. The purpose of adding a version to the validation info is to provide a mechanism to save validation info snapshots in time as well as for uniqueness of file paths. The logic will be as follows, if the current version is greater than or equal to a saved validation logits version (largest that is less than or equal to the current version), pick up that version.

### CODE REVIEW

File: aiu_fms_testing_utils/testing/validation.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 import torch
 from aiu_fms_testing_utils.utils.aiu_setup import dprint
 import os
+from aiu_fms_testing_utils._version import version_tuple
```
Comment (ani300) [inline aiu_fms_testing_utils/testing/validation.py:7]: reorder imports here

File: tests/testing/test_validation.py [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
     extract_validation_information,
     load_validation_information,
+    get_validation_info_path,
+    find_validation_info_path,
 )
+from aiu_fms_testing_utils._version import version_tuple
 from fms.models import get_model
 from fms.utils.generation import pad_input_ids
 import torch
+from pathlib import Path
```
Comment (ani300) [inline tests/testing/test_validation.py:15]: reorder imports here

**Comment by JRosenkranz** [comment]
bot:test
TEST_FILE=test_decoders.py MODEL_ID=ibm-granite/granite-3.3-8b-instruct BATCH_SIZE=1 SEQUENCE_LENGTH=2048 USE_TINY_MODEL=1
